source/language/pulses.rst

```diff
@@ -108,10 +108,12 @@ Inline calibration blocks
 
 As calibration grammars may require the ability to insert top-level configuration information, perform program setup, or make inline calls
 to calibration-level instructions, OpenQASM supports the ability to declare a ``cal`` block. Within the ``cal`` block the
-semantics of the selected calibration grammar are valid. The ``cal`` block is of the same scope level as the enclosing block. The calibration
-grammar implementer may therefore choose to allow referencing or modifying values outside of the ``cal`` block but within the containing scope.
-Values declared within the ``cal`` block are only referenceable from other ``cal`` blocks or ``defcal`` declarations that may observe that scope as defined
-by the calibration grammar implementer. Values may not leak back to the enclosing blocks scope.
+semantics of the selected ``defcalgrammar`` are valid. The ``cal`` block is of the same scope level as the enclosing block. The
+calibration grammar may choose to allow capturing values (with chosen syntax) from within the ``cal``
+block that were declared within the containing parent scope.
+The calibration grammar may therefore choose to allow capturing values declared within the parent scope.
```

Comment:
Duplicate sentences? 

> The calibration grammar may choose to allow capturing values (with chosen syntax) from within the ``cal``
block that were declared within the containing parent scope. The calibration grammar may therefore choose to allow capturing values declared within the parent scope.

Reply:
Fixed in 0cabc5f.